label_studio/data_import/uploader.py

```diff
@@ -35,29 +33,34 @@ def is_binary(f):
 
 
 def csv_generate_header(file):
-    """ Generate column names for headless csv file """
+    """Generate column names for headless csv file"""
     file.seek(0)
     names = []
     line = file.readline()
 
     num_columns = len(line.split(b',' if isinstance(line, bytes) else ','))
     for i in range(num_columns):
-        names.append('column' + str(i+1))
+        names.append('column' + str(i + 1))
     file.seek(0)
     return names
 
 
 def check_max_task_number(tasks):
     # max tasks
     if len(tasks) > settings.TASKS_MAX_NUMBER:
-        raise ValidationError(f'Maximum task number is {settings.TASKS_MAX_NUMBER}, '
-                              f'current task number is {len(tasks)}')
+        raise ValidationError(
+            f'Maximum task number is {settings.TASKS_MAX_NUMBER}, '
+            f'current task number is {len(tasks)}'
+        )
 
 
 def check_tasks_max_file_size(value):
     if value >= settings.TASKS_MAX_FILE_SIZE:
-        raise ValidationError(f'Maximum total size of all files is {settings.TASKS_MAX_FILE_SIZE} bytes, '
-                              f'current size is {total} bytes')
+        raise ValidationError(
+            f'Maximum total size of all files is {settings.TASKS_MAX_FILE_SIZE} bytes, '
+            f'current size is {value} bytes'
```

Comment:
This PR looks like a lot of changes, but you can go commit-by-commit through and see that it is really just

- fix the bug
- remove unused imports
- autoformat
- add tests